companies/exotargets.json

```diff
@@ -6,10 +6,10 @@
     "name": "eXotargets Data Network GmbH",
     "address": "Hafenstr. 3\n60327 Frankfurt am Main\nGermany",
     "phone": "+49 69 29921932",
-    "email": "dsb-exo@ibs-data-protection.de",
+    "email": "datenschutzanfrage@exotargets.de",
```

Comment:
nice! Since this email is obviously only for privacy requests, you can set suggested-transport-medium to email as well :)

Reply:
This is already the case, see line 15.

Reply:
Oops, sorry.